app/Offer/OfferServiceProvider.php

```diff
@@ -566,6 +568,14 @@ public function register(): void
             fn () => new UpdateWorkflowStatusRequestHandler($container->get('event_command_bus'))
         );
 
+        $container->addShared(
+            GetContributorsRequestHandler::class,
```

Comment:
As in https://github.com/cultuurnet/udb3-backend/pull/1253/files#r1099961153, I think this class can be auto-wired if you define the contributor repository in the service container using its interface name instead of its implementation name.

Reply:
This is unfortunatly no longer relevant since I have to use non FQN definitions.

Reply:
Indeed, now that you have to inject `offer_permission_voter` you cannot use auto-wiring.